docs/sources/configure-server/reference-configuration-parameters/index.md

```diff
@@ -846,7 +1023,16 @@ The `query_frontend` block configures the query-frontend.
 # IP address to advertise to the querier (via scheduler) (default is
 # auto-detected from network interfaces).
 # CLI flag: -query-frontend.instance-addr
-[address: <string> | default = ""]
+[instance_addr: <string> | default = ""]
```

Comment:
I bet many people have been caught by this unfortunate inconsistency: we use `instance_addr` everywhere else. In the CLI, it's actually `instance-addr`, what makes things even more confusing.

To prevent any future confusion, I’m changing address to `instance_addr` while maintaining backward compatibility with the previous version.